neural_compressor/conf/pythonic_config.py

```diff
@@ -28,6 +28,8 @@ def __init__(self,
                  inputs=[],
                  outputs=[],
                  backend='default',
+                 domain='auto',
```

Comment:
why changing here? this should be legacy conf code, right?

Reply:
> 
it maps new config items to the legacy conf, so the legacy conf should have these attributes. Keep these attrs but don't take them as initialization params is ok?